### PR TITLE
Add readme.md to sub folders - tabular, text and vision

### PR DESCRIPTION
**Pull request by vinuthakaranth**
# Description
Add readme.md to sub folders - tabular, text and vision

# Checklist

- [x] I have read the contribution guidelines.
  - [General](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
  - [SDK](https://github.com/Azure/azureml-examples/blob/main/sdk/CONTRIBUTING.md)
  - [CLI](https://github.com/Azure/azureml-examples/blob/main/cli/CONTRIBUTING.md)
- [ ] I have coordinated with the docs team (mldocs@microsoft.com) if this PR deletes files or changes any file names or file extensions.
- [ ] Pull request includes test coverage for the included changes.
- [ ] This notebook or file is added to the [CODEOWNERS](https://github.com/Azure/azureml-examples/blob/main/.github/CODEOWNERS) file, pointing to the author or the author's team.

### CODE REVIEW

**Review by imatiach-msft — APPROVED** [review]

File: sdk/python/responsible-ai/text/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+| Text Question Answering | [Stanford Question Answering Dataset (SQuAD)](https://huggingface.co/datasets/squad) | Text | Explanation, Error Analysis | [responsibleaidashboard-text-question-answering-squad.ipynb](./text/responsibleaidashboard-text-question-answering-squad.ipynb) | [Text Dashboard Generation](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-responsible-ai-text-dashboard?view=azureml-api-2)|
+
+
+To learn more about the different types of Dashboard visit the tutorial -
+[Text Dashboard Generation](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-responsible-ai-text-dashboard?view=azureml-api-2)
+
+
+## Supportability 🧰
+Currently, we support datasets having numerical and categorical features. The following table provides the scenarios supported for each of the four responsible AI components:
```
Comment (gaugup) [inline sdk/python/responsible-ai/text/README.md:37]: Is this true for text dashboards? I guess not. Maybe remove it.

File: sdk/python/responsible-ai/vision/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+| Image Object Detection | [Object Detection Fridge Images](https://github.com/microsoft/computervision-recipes/tree/master/scenarios/detection) | Image | Explanation, Error Analysis | [responsibleaidashboard-object-detection-MSCOCO.ipynb](./vision/responsibleaidashboard-object-detection-MSCOCO.ipynb) | [Vision Dashboard Generation](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-responsible-ai-image-dashboard?view=azureml-api-2) |
+
+
+To learn more about the different types of Dashboard visit the tutorial -
+[Vision Dashboard Generation](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-responsible-ai-image-dashboard?view=azureml-api-2)
+
+
+## Supportability 🧰
+Currently, we support datasets having numerical and categorical features. The following table provides the scenarios supported for each of the four responsible AI components:
```
Comment (gaugup) [inline sdk/python/responsible-ai/vision/README.md:37]: Is this true for vision dashboards? I guess not. Maybe remove it.

File: sdk/python/responsible-ai/text/README.md (hunk 1 above)
Comment (imatiach-msft) [reply sdk/python/responsible-ai/text/README.md:37]: yes any additional metadata features with categorical and numeric values are supported - but the dashboard itself runs on freeform text

File: sdk/python/responsible-ai/vision/README.md (hunk 2 above)
Comment (imatiach-msft) [reply sdk/python/responsible-ai/vision/README.md:37]: yes any additional metadata features with categorical and numeric values are supported - but the dashboard itself runs on images

**Review by gaugup — APPROVED** [review]

**Pull request merged**